pubsub/pubsub.go

```diff
@@ -33,18 +33,18 @@ type PubSub interface {
 // BulkMessager is the interface defining BulkPublish and BulkSubscribe definitions for message buses
 type BulkMessager interface {
```

Comment:
Should this be "BulkMessenger"? Or "BulkPublisher"?

Reply:
Ok. How about "Messenger" tho?

Reply:
I am OK with that. So`BulkMessenger` interface?

Reply:
Segregated in two different interfaces to let BulkPublish and BulkSubscribe remain independent.